packages/server-side-render/src/hook.ts

```diff
@@ -5,17 +5,25 @@ import { debounce } from '@wordpress/compose';
 import { useEffect, useState, useRef } from '@wordpress/element';
 import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
+
+// @ts-expect-error __experimentalSanitizeBlockAttributes is not typed
 import { __experimentalSanitizeBlockAttributes } from '@wordpress/blocks';
 
-export function rendererPath( block, attributes = null, urlQueryArgs = {} ) {
+export function rendererPath(
+	block: string,
+	attributes: Record< string, unknown > | null = null,
+	urlQueryArgs: Record< string, unknown > = {}
+): string {
 	return addQueryArgs( `/wp/v2/block-renderer/${ block }`, {
 		context: 'edit',
 		...( null !== attributes ? { attributes } : {} ),
 		...urlQueryArgs,
 	} );
 }
 
-export function removeBlockSupportAttributes( attributes ) {
+export function removeBlockSupportAttributes(
+	attributes: Record< string, unknown >
```

Comment:
Instead of making an assertion below for `attributes?.style`, we can type it here

```suggestion
	attributes: Record< string, unknown > & {
		style?: Record< string, unknown >;
	}
```